From fca87976136f2256c970e950b3d7ffa51dcc849f Mon Sep 17 00:00:00 2001
From: Sanjeev Gupta <ghane0@gmail.com>
Date: Fri, 2 Dec 2016 01:10:47 +0800
Subject: [PATCH 107/268] All internal references (anchors) clean

sanjeev@X201wily:~/SRC/ntpsec$ ./waf linkcheck
206 anchors, 46 references, 0 references unsatisfied

'linkcheck' finished successfully (0.100s)
---
 docs/discipline.txt              | 2 +-
 docs/includes/assoc-commands.txt | 1 +
 docs/includes/clock-options.txt  | 4 +++-
 docs/includes/clockopt.txt       | 2 +-
 docs/miscopt.txt                 | 2 ++
 5 files changed, 8 insertions(+), 3 deletions(-)

diff --git a/docs/discipline.txt b/docs/discipline.txt
index 97433d1..3ec6c22 100644
--- a/docs/discipline.txt
+++ b/docs/discipline.txt
@@ -176,7 +176,7 @@ cluster algorithms to refine and validate the initial clock offset
 estimate. Normally, this takes several updates before setting the system
 clock. As the default minimum poll interval in most configurations is
 about one minute, it can take several minutes before setting the system
-clock. The +iburst+ option of the link:confopt.html#burst[+server+]
+clock. The +iburst+ option of the link:assoc.html#burst[+server+]
 command changes the behavior at restart and is recommended for
 client/server configurations. When this option is enabled, the client
 sends a volley of six requests at intervals of two seconds. This usually
diff --git a/docs/includes/assoc-commands.txt b/docs/includes/assoc-commands.txt
index 7c5c134..2fea603 100644
--- a/docs/includes/assoc-commands.txt
+++ b/docs/includes/assoc-commands.txt
@@ -17,6 +17,7 @@ link-local IPV6 address with an interface specified in
   and the mode in which to operate. The _address_ can be either a DNS
   name or an IP address in dotted-quad notation.
 
+[[pool]]
 +pool+::
   For server addresses, this command mobilizes a persistent client
   mode association with a number of remote servers. In this mode the
diff --git a/docs/includes/clock-options.txt b/docs/includes/clock-options.txt
index ef44b89..7326c5d 100644
--- a/docs/includes/clock-options.txt
+++ b/docs/includes/clock-options.txt
@@ -1,9 +1,11 @@
 // Options for refclocks.  Included twice.
 
+[[options]]
 +refclock+ _drivername_ [+unit+ _u_] [+prefer+] [+subtype+ _int_] [+mode+ _int_] [+minpoll+ _int_] [+maxpoll+ _int_] [+time1+ _sec_] [+time2+ _sec_] [+stratum+ _int_] [+refid+ _string_] [+path+ 'filename'] [+ppspath+ 'filename'] [+baud+ 'number'] [+flag1+ {+0+ | +1+}] [+flag2+ {+0+ | +1+}] [+flag3+ {+0+ | +1+}] [+flag4+ {+0+ | +1+}]::
   This command is used to configure reference clocks.
   The required _drivername_ argument is the shortname of a driver type
-  (e.g. +shm+, +nmea+, +generic+; see the "Reference Clock Drivers"
+  (e.g. +shm+, +nmea+, +generic+;
+  see the link:refclock.html [Reference Clock Drivers]
   page for a full list. The options are interpreted as follows:
 
   +unit+;;
diff --git a/docs/includes/clockopt.txt b/docs/includes/clockopt.txt
index 2a59a70..34bde1e 100644
--- a/docs/includes/clockopt.txt
+++ b/docs/includes/clockopt.txt
@@ -1,5 +1,5 @@
 === Reference Clock Commands and Options ===
+* link:clockopt.html#addrs[refclock - specify reference clock server]
 * link:clockopt.html#options[options - specify clock-driver options]
-* link:clockopt.html#server[server - specify reference clock server]
 * link:comdex.html[Command Index]
 
diff --git a/docs/miscopt.txt b/docs/miscopt.txt
index 4199fa7..59d0e39 100644
--- a/docs/miscopt.txt
+++ b/docs/miscopt.txt
@@ -22,6 +22,7 @@ include::includes/misc-options.txt[]
 // A subset of the tos options is described on the ntp.conf manual page.
 // If you change this, be very sure to keep that synchronized.
 
+[[tos]]
 +tos+ [+beacon+ 'beacon' | +ceiling+ 'ceiling' | +cohort+ {+0+ | +1+} | +floor+ 'floor' | +maxclock+ 'maxclock' | +maxdist+ 'maxdist' | +minclock+ 'minclock' | +mindist+ 'mindist' | +minsane+ 'minsane' | +orphan+ 'stratum' | +orphanwait+ 'delay']::
   This command alters certain system variables used by the clock
   selection and clustering algorithms. The default values of these
@@ -95,6 +96,7 @@ include::includes/misc-options.txt[]
     and selectable before using backup sources, and avoids transient use
     of the backup sources at startup.
 
+[[ttl]]
 +ttl+ 'hop ...'::
   This command specifies a list of TTL values in increasing order. up to
   8 values can be specified. In manycast mode these values are used in
-- 
2.7.4

